src/components/organisms/newsDetail.php

```diff
@@ -1,34 +1,37 @@
 <?php
 
-require_once __DIR__ .'/breadcrumb.php';
+require_once __DIR__ .'/../molecules/breadcrumb.php';
+require_once __DIR__ .'/../molecules/newsEdit.php';
+require_once __DIR__ .'/../molecules/newsActions.php';
+require_once __DIR__ .'/../molecules/newsView.php';
 
 class NewsDetail {
     /**
-     * ニュース詳細をレンダリング
+     * ニュース詳細コンポーネントをレンダリング
      *
-     * @param PostsDTO $post
+     * @param PostsDTO $post 投稿DTO
+     * @param string $mode 表示モードか、編集モードか (固定値: MODE_VIEW, MODE_EDIT, MODE_NEW)
      * @return void
      */
-    public static function render(PostsDTO $post): void {
-        $props = [['name' => 'ニュース - '. $post->title, 'link' => $_SERVER['REQUEST_URI']]];
+    public static function render(PostsDTO $post, string $mode): void {
+        $newsDetail = new NewsDetail(['post' => $post, 'mode' => $mode]);
+        $editorMode = in_array($mode, [MODE_EDIT, MODE_NEW]);
```

Comment:
in_arrayを使うなら、`true`を第三引数として追加すると型もチェックするのでこれはおすすめです。
https://www.php.net/manual/en/function.in-array.php
```suggestion
        $editorMode = in_array($mode, [MODE_EDIT, MODE_NEW], true);
```